### PR TITLE
[LOOP-4483] Study Product Selection

### PR DESCRIPTION
**Pull request by Camji55**
https://tidepool.atlassian.net/browse/LOOP-4483

### CODE REVIEW

File: TidepoolOnboarding/Previews/OnboardingViewModel+Preview.swift [hunk 1]
```diff
@@ -16,6 +16,7 @@ extension OnboardingViewModel {
 fileprivate class PreviewOnboardingProvider: OnboardingProvider {
     
     var allowDebugFeatures: Bool = true
+    var studyProduct: StudyProduct = .none
```
Comment (nhamming) [inline TidepoolOnboarding/Previews/OnboardingViewModel+Preview.swift:19]: OK. I see the LoopKit study product enum is needed here. Would it be possible to define it here? It seems only Loop and TidepoolOnboarding need it, but maybe I'm overlooking something. 

I guess I don't see the need for the definition in LoopKit, but let me know.

**Review by nhamming — COMMENTED** [review]
just a question about scoping

**Comment by Camji55** [comment]
Oops...

File: TidepoolOnboarding/Previews/OnboardingViewModel+Preview.swift (hunk 1 above)
Comment (Camji55) [reply TidepoolOnboarding/Previews/OnboardingViewModel+Preview.swift:19]: I moved the enum to Tidepool Onboarding.

**Review by nhamming — APPROVED** [review]
LGTM

**Pull request merged**